### DISCUSSION TITLE
I have a question about image metadata.

### DISCUSSION THREAD

**Discussion by LittleDevHo** [SupaGruen/SD-CheatSheet#7] [General]
When I insert an image file, it doesn't generate metadata for all the images, is there a reason for this?

**Comment by SupaGruen** [comment]
Are these images generated with Stable Diffusion? What's the output? Is it "No EXIF data detected" or nothing at all?
I added an image with metadata, please [check this one](https://supagruen.github.io/StableDiffusion-CheatSheet/img/other/metadata-example.jpeg) to see how it works.
